kmod/src/inode.c

```diff
@@ -1595,10 +1603,15 @@ int scoutfs_scan_orphans(struct super_block *sb)
 			ret = -ENOENT;
 			break;
 		}
+
+		deleted = true;
```

Comment:
I think we should only return 1 when we've made progress by delete_inode_items() returning success.  We probably don't want it to spin constantly if the list only contains failures?  It's a tricky case.

Reply:
yeah this was bad logic left over from a previous version.